pimpmycause/profiles/migrations/0019_auto_20170215_1930.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
it had the slugfield added to the model. I removed that field and re-run migrations